From a67b0992b081a5696eb7a77b027563ac12674098 Mon Sep 17 00:00:00 2001
From: Hiroshi Shirosaki <h.shirosaki@gmail.com>
Date: Wed, 9 Jul 2014 17:10:26 +0900
Subject: [PATCH 39/41] Fix include isl/stdint.h error

---
 gcc/config.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/gcc/config.in b/gcc/config.in
index 27497d4..d44fb98 100644
--- a/gcc/config.in
+++ b/gcc/config.in
@@ -2053,3 +2053,6 @@
 #undef vfork
 #endif
 
+#if !defined HAVE_STDINT_H && !defined _ISL_INCLUDE_ISL_STDINT_H
+#define _ISL_INCLUDE_ISL_STDINT_H
+#endif
-- 
2.0.0.msysgit.0

